fenix/app/src/main/java/org/mozilla/fenix/home/toolbar/SearchSelectorBinding.kt

```diff
@@ -76,4 +79,8 @@ class SearchSelectorBinding(
                 binding.searchSelectorButton.setIcon(icon, name)
             }
     }
+
+    companion object {
+        const val TAP_INCREASE_DPS_FOR_HEIGHT = 10
```

Comment:
Nit: I think we could find a more suggestive name for this, something like

```suggestion
        const val SEARCH_SELECTOR_INCREASE_HEIGHT_DPS = 10
```